config/config.exs

```diff
@@ -30,4 +30,17 @@ config :membrane_opentelemetry, enabled: telemetry_enabled
 
 config :logger, :console, metadata: [:room, :peer]
 
+config :membrane_rtc_engine_timescaledb, ecto_repos: [Membrane.RTC.Engine.TimescaleDB.Repo]
+
+config :membrane_rtc_engine_timescaledb, Membrane.RTC.Engine.TimescaleDB.Repo,
+  database: "membrane",
+  username: "postgres",
+  password: "postgres",
+  hostname: "localhost",
+  port: 5432,
```

Comment:
I am not sure if this should be hardcoded. Shouldn't we pass this config via environment variables?